OWNERS

```diff
@@ -1,15 +1,22 @@
+component: "Monitoring"
+
 reviewers:
 - brancz
-- mxinden
-- elad661
-- ironcladlou
-- squat
 - s-urbaniak
+- paulfantom
+- LiliC
```

Comment:
lower case :) lilic and thanks!

Reply:
This is a copy-paste from [master](https://github.com/openshift/cluster-monitoring-operator/blob/master/OWNERS#L16) so you might want to fix this there too :)

Reply:
Will do! Thanks!